cognite/seismic/protos/v1/seismic_service_messages.proto

```diff
@@ -37,13 +37,19 @@ message SearchSurveysRequest {
     CoverageParameters include_coverage = 5;    // set this field to include coverage in the response (default: false)
     bool include_grid_transformation = 6;    // set this field to include the manually specified grid transformation in the response (default: false)
     bool include_custom_coverage = 7;    // set this field to include the customer-provided survey coverage in the response
```

Comment:
Hmm, now this is obsolete.

Reply:
I'm going through the code and i think the difference is when including the custom coverage, it will always return with the survey's crs, whereas the outer coverage will transform the geometry with the crs provided in the coverage parameters